gradoop-common/src/main/java/org/gradoop/common/util/AsciiGraphLoader.java

```diff
@@ -47,9 +47,9 @@
 public class AsciiGraphLoader<G extends GraphHead, V extends Vertex, E extends Edge> {
 
   /**
-   * Gradoop configuration
+   * Factory provider for EPGM elements.
```

Comment:
```suggestion
   * Factory provider for graph elements.
```
Please also remove "EPGM" wording if it is generalized like in this class